misc/glossary.md

```diff
@@ -0,0 +1,156 @@
+# Glossary
+
+
+  
+## Current terms
+
+**Administrative Team:** Team that directly interfaces with SubDAO governance, makes proposals, receives budgets and has governance privileges.
```

Comment:
I think this is deprecated? Not 100% sure.

Maybe we need sections as: 'Current' 'Probably Deprecated' 'Deprecated' lol.